<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Administrator</class>
 <widget class="QWidget" name="Administrator">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>700</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Администратор</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="windowFilePath">
   <string/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1801</width>
     <height>831</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>UpArrowCursor</cursorShape>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Контроллеры</string>
    </attribute>
    <widget class="QTableView" name="tableViewControllers">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>1111</width>
       <height>591</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxSoftWare">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>188</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxType">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>248</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxMode">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>158</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxProgrammer">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>218</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditDate">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>40</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditControllerID">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>10</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditBoardNumber">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>70</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditSensorNumber">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>100</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit70027Number">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>130</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditSDCardNumber">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>280</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelControllerID">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>10</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер контроллера</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelDate">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>40</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelBoardNumber">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>70</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер платы</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSensorNumber">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>100</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер датчика</string>
     </property>
    </widget>
    <widget class="QLabel" name="label70027Number">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>130</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер 70027 платы</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelMode">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>160</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Модификация</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSoftWare">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>190</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Версия ПО</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelProgrammer">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>220</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Сотрудник</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelType">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>250</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Тип</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSDCardNumber">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>280</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер SD карты</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatus">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>460</y>
       <width>311</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Статус: Всё хорошо</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonEnter">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>480</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRename">
     <property name="geometry">
      <rect>
       <x>1240</x>
       <y>480</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Заменить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRemove">
     <property name="geometry">
      <rect>
       <x>1350</x>
       <y>480</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>1450</x>
       <y>30</y>
       <width>21</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout"/>
    </widget>
    <widget class="QCalendarWidget" name="calendarWidget">
     <property name="geometry">
      <rect>
       <x>1133</x>
       <y>510</y>
       <width>321</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditClient">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>310</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditSetupPlace">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>340</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditReason">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>370</y>
       <width>141</width>
       <height>81</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labeClient">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>310</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Заказчик</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelSetupPlace">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>340</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Место установки</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelReason">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>370</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Примечание</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Платы</string>
    </attribute>
    <widget class="QTableView" name="tableViewPlates">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>1071</width>
       <height>591</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="textPlateEditDate">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>60</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlateDate">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>60</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAdd">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>350</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEditPlateSearch">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlateNumber">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>30</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер платы</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textPlateNumber">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>30</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlateVersion">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>90</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Версия</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textPlateVersion">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>90</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="textPlateName">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>120</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlateName">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>120</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Название</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlateUser">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>150</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Прошивальщик</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textPlateSettings">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>180</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlateSettings">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>180</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Настройка</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textPlateErrors">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>210</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlateErrors">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>210</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Ошибки</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlateLock">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>240</y>
       <width>131</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Защита</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMove">
     <property name="geometry">
      <rect>
       <x>1240</x>
       <y>350</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Изменить</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonPlateRemove">
     <property name="geometry">
      <rect>
       <x>1350</x>
       <y>350</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Удалить</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxLock">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>240</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Lock</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Unlock</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="comboBoxUser">
     <property name="geometry">
      <rect>
       <x>1300</x>
       <y>150</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelPlateStatus">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>300</y>
       <width>311</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Статус: Всё хорошо</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonRed">
     <property name="geometry">
      <rect>
       <x>1130</x>
       <y>390</y>
       <width>101</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Перекрасить</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
